addons/mail/static/tests/suggestion/suggestion.test.js

```diff
@@ -330,3 +337,29 @@ test("Mention with @everyone", async () => {
     await click(".o-mail-Composer-suggestion");
     await contains(".o-mail-Composer-input", { value: "@everyone " });
 });
+
+test("@everyone mention adds channel members to the message recipients", async () => {
```

Comment:
As discussed, why not just add an extra step at the end of `Mention with @everyone` to verify the message is mentioned for the current user?

That would be good enough, no need to add an extra test.